pipeline.lisp

```diff
@@ -36,7 +36,10 @@ As such, (1 4) matches the following:
   (:method ((pipeline pipeline) (place list))
     (find-place (pipeline pipeline) place))
   (:method ((pipeline pipeline) (name symbol))
-    (find-place (pipeline pipeline) (gethash name (names pipeline)))))
+    (multiple-value-bind (path found)
+        (gethash name (names pipeline))
+      (when found
+        (find-place (pipeline pipeline) path)))))
```

Comment:
It's been a while since I looked at this library, but can't it just be `(or (gethash name (names pipeline)) (find-place (pipeline pipeline) path))`?

Reply:
Not entirely sure what you mean. It seems that what you wrote would *either* return the path (ie. the sequence of ints) if `NAME` was found in `NAMES`, *or* it would try to use an unbound variable (since it no longer binds `PATH` presumably).